src/app/shared/services/authentication/authentication.service.ts

```diff
@@ -8,8 +8,8 @@ import { environment } from '@environment';
 })
 
 export class AuthenticationService extends BaseService {
-  public signIn(endpoint: string, params: any): Observable<any> {
-    const apiUrl = `${environment.apiBaseUrl}/${endpoint}`;
+  signIn(params: any): Observable<any> {
+    const endpoint = 'oauth/token';
```

Comment:
As explained above, prefer adding the version here:

```suggestion
    const endpoint = 'v1/oauth/token';
```

There will be some repetition but it's more flexible.